contrib/objcwrapper/src/objc_model.nit

```diff
@@ -19,6 +19,33 @@ module objc_model
 class ObjcModel
 	# All analyzed classes
 	var classes = new Array[ObjcClass]
+
+	# Is `objc_type` known by this model?
+	fun knows_type(objc_type: Text): Bool
+	do
+		for c in classes do
+			if c.name == objc_type then return true
```

Comment:
I thought of making `classes` a `HashMap[String, ObjcClass]` but I was worried of name collisions when parsing many files at once. This problem may be at another level though.